client/src/document/ingredients/browser-compatibility-table/index-desktop.scss

```diff
@@ -32,13 +32,13 @@
     }
   }
 
-  .table-container {
+  .bc-table-outer {
     margin: 0 -3rem;
     overflow: auto;
-    width: 100vw;
+    width: 100vw !important;
```

Comment:
My personal opinion, don't need to overwrite width because `.table-container` width is 100% applying to all tables. And we don't need view port width. Remove it.